pkg/controller/monitoring.go

```diff
@@ -120,6 +121,49 @@ func (c *Controller) updateHealthStatusForTenant(tenant *miniov2.Tenant) error {
 	if err != nil {
 		// show the error and continue
 		klog.Infof("'%s/%s' Failed to get cluster health: %v", tenant.Namespace, tenant.Name, err)
+		if strings.Contains(err.Error(), "failed to verify certificate") {
+			externalCertSecret := tenant.Spec.ExternalCertSecret
+			klog.Info("Let's check if there is an external cert for the tenant...")
+			if externalCertSecret != nil {
+				// Check that there is a secret that starts with "operator-ca-tls-" to proceed with the renewal
+				secretsAvailableAtOperatorNS, _ := c.kubeClientSet.CoreV1().Secrets(miniov2.GetNSFromFile()).List(context.Background(), metav1.ListOptions{})
+				for _, secret := range secretsAvailableAtOperatorNS.Items {
```

Comment:
if there's an error `secretsAvailableAtOperatorNS ` will be null and this would cause a null pointer exception.